### DISCUSSION TITLE
[Feature] Filter inside album

### DISCUSSION THREAD

**Discussion by VibrantTreeStump** [immich-app/immich#11716] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I organize files from group events like this:
  - Each event is an album
  - Each file retains its original filename prefixed with "name_" where name is the person who took the photo/video

Now if I want to search for all pictures I took while on holiday with friends, I would like to be able to go into the album and then search for files starting with "myName_". I haven't found a way to do this with the global search.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2407
